static/docs/get-started/connect-code-and-data.md

```diff
@@ -48,17 +48,19 @@ recommend creating a virtual environment with a tool such as
 $ virtualenv -p python3 .env
 $ source .env/bin/activate
 $ echo ".env/" >> .gitignore
-$ pip install -r requirements.txt
+$ pip install -r src/requirements.txt
```

Comment:
check other places in command references where we use example-get-started repo, sometimes we provide instruction on how to install it in the details section

Reply:
Another good catch, thanks! Fixed in 9885f13